tubular/imputers.py

```diff
@@ -48,7 +54,7 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         X = super().transform(X)
 
         for c in self.columns:
-            X[c] = X[c].fillna(self.impute_values_[c])
+            X = X.with_column(X[c].fill_null(self.impute_values_[c]))
```

Comment:
S: might need to add .alias(c) on the end? Not sure, maybe it does work without though.. 
`X = X.with_columns(X[c].fill_null(self.impute_values_[c]).alias(c))`